provider/pdns/pdns.go

```diff
@@ -72,23 +72,19 @@ type PDNSConfig struct {
 
 // TLSConfig is comprised of the TLS-related fields necessary to create a new PDNSProvider
 type TLSConfig struct {
-	TLSEnabled            bool
+	SkipTLSVerify         bool
 	CAFilePath            string
 	ClientCertFilePath    string
 	ClientCertKeyFilePath string
 }
 
 func (tlsConfig *TLSConfig) setHTTPClient(pdnsClientConfig *pgo.Configuration) error {
-	if !tlsConfig.TLSEnabled {
+	if tlsConfig.SkipTLSVerify {
 		log.Debug("Skipping TLS for PDNS Provider.")
```

Comment:
This doesn't seem right. It still has to configure TLS, it just needs to configure it to skip certificate verification.